pkg/build/strategies/sti/sti.go

```diff
@@ -345,7 +369,14 @@ func (builder *STI) Prepare(config *api.Config) error {
 
 	// fetch sources, for their .s2i/bin might contain s2i scripts
 	if len(config.Source) > 0 {
-		if builder.sourceInfo, err = builder.source.Download(config); err != nil {
+		timeStart := time.Now()
+		builder.sourceInfo, err = builder.source.Download(config)
+		builder.result.BuildInfo.StepInfo = utilstatus.AddStepInfo(
+			builder.result.BuildInfo.StepInfo,
+			api.GatherInputContentStep,
+			timeStart,
+		)
```

Comment:
not needed.  the "source.Download" that is performed when invoking s2i from openshift is just a file copy:
https://github.com/openshift/source-to-image/blob/master/pkg/scm/file/download.go#L21

because openshift has already done the clone operation.

and it's probably not worth calculating the filecopy time.